packages/popover/src/Popover/Popover.hooks.tsx

```diff
@@ -48,9 +52,13 @@ export function usePopoverContextProps({
     | 'refEl'
   >
 >) {
+  const { forceUseTopLayer } = useMigrationContext();
   const context = usePopoverContext();
-  const renderMode =
-    renderModeProp || context.renderMode || RenderMode.TopLayer;
+  const renderMode = getRenderMode({
+    forceUseTopLayer,
+    renderModeProp,
+    renderModeCtxVal: context.renderMode,
+  });
```

Comment:
I'm wondering if this helper method is providing much given that we have to pass in 2 of the 3 possible options anyway. It creates the need to have to jump to another file to see what it's doing and I'm not sure it's more readable than just having the logic inline:
```ts
const renderMode = forceUseTopLayer ? 
  RenderMode.TopLayer : 
  renderModeProp || context.renderMode || RenderMode.TopLayer;
```
Mostly nit, but curious to hear your thoughts